analyses/survival-analysis/util/ggforest_new.R

```diff
@@ -0,0 +1,149 @@
+# this is a function modified from the original ggforest 
+# It has two modifications:
+# 1. change the y axis range (main reason why plots don't plot is because the ranges are too big to be plotted)
+# 2. from the coefficient table, remomve any covaraites that are inf/NA so that they do not get plotted
+
+ggforest_new <- function(model, data = NULL,
```

Comment:
I'm very concerned about this approach - "from the coefficient table, remomve any covaraites that are inf/NA so that they do not get plotted." (sic)

We do not want to be cherry-picking results. If a model is returning `NA` or `Inf`, that is meaningful information about either the data going into the model, model convergence, adherence to model assumptions, etc. Removing this information is therefore hiding the fact that there is a potential problem with the model, which is not transparent for reporting results.

In addition, this code looks almost entirely derived from the `survminer` function `ggforest` but there is no acknowledgement given consistent with the `survminer` GPL-2 license. The source must be appropriately acknowledged so there is no violation of licensing terms.